src/zappy_ai_src/zappy_ai

```diff
@@ -6,7 +6,7 @@ import argparse
 
 def displayHelp():
     """ Display the Usage of the Zappy AI """
-    print("USAGE: ./zappy_ai -p port -n name -h machine")
+    print("USAGE: ./zappy_ai_src -p port -n name -h machine")
```

Comment:
This is still zappy_ai

Reply:
This will be modified by the merge, no need to modify it. Also you should consider using:
```suggestion
    print("USAGE: ./" + sys.argv[0] + " -p port -n name -h machine")
```
instead of the binary name in plain text @31Nathan 

@LucasTesnier Since `zappy_ai` was modified into `zappy_ai_src` I suspect that there a faulty use of search and replace. Could you double check that nothing broke there ?

Reply:
No problem detected